book/src/utilities/events.md

```diff
@@ -1,13 +1,16 @@
 # Events
 
-There is a simple way to fire and respond to events.
+There is a simple way to fire and respond to global events.
 
 ```rust,noplaypen
-struct MyEvent(u64);
+pub use hobo::{
+	prelude::*,
+	create as e,
+};
 
-// -- snip --
```

Comment:
this was to show that there's an unspecified amount of code between the event definition and its use, idk maybe it wasn't important